components/eam/src/physics/cam/cloud_diagnostics.F90

```diff
@@ -155,9 +155,9 @@ subroutine cloud_diagnostics_init()
     call addfld ('TGCLDCWP',horiz_only,    'A',wpunits,'Total grid-box cloud water path (liquid and ice)', &
          sampling_seq=sampling_seq)
     call addfld ('TGCLDLWP',horiz_only,    'A',wpunits,'Total grid-box cloud liquid water path', &
-         sampling_seq=sampling_seq)
+         sampling_seq=sampling_seq, standard_name='atmosphere_mass_content_of_cloud_condensed_water')
```

Comment:
This standard name should be "atmosphere_mass_content_of_cloud_liquid_water", in contrast with cloud_ice  below.

Reply:
Yep. I meant to put that on TGCLDCWP.